sdk/typescript/runtime/main.go

```diff
@@ -263,6 +261,8 @@ func (t *TypescriptSdk) installedSDK(ctr *Container, runtime SupportedTSRuntime)
 		return ctr.WithExec([]string{"bun", "install", "--no-verify", "--no-progress", "--summary"}).Directory(ModSourceDirPath)
 	case Node:
 		return ctr.
+			// Enable corepack so we can use yarn v4 which is supposed to be faster than npm or yarn v1.
+			WithExec([]string{"corepack", "enable"}).
 			WithExec([]string{"yarn", "set", "version", "stable"}).
 			WithExec([]string{"yarn", "workspaces", "focus", "--production"}).Directory(ModSourceDirPath)
```

Comment:
The `yarn set version stable` is what is actually enabling `yarn v4`.  
That may cause issues since it uses the `pnp` linker by default.

You also probably want to change the `yarn workspaces focus --production` to just use `yarn install`

Reply:
Hmm I'm only use it for the TS SDK and it worked well so far :O

Reply:
It is using `pnp` though.